src/test/librbd/fsx.cc

```diff
@@ -1164,7 +1164,7 @@ krbd_resize(struct rbd_ctx *ctx, uint64_t size)
 {
 	int ret;
 	int count = 0;
-	uint64_t effective_size;
+	uint64_t effective_size{};
```

Comment:
This is a style nit so feel free to ignore but initializing integers this way is not explicit and even weird IMO.  I'd stick to conventional ` = 0;`.

Reply:
Meta comment: C++ initialization rules are famously confusing and complex. We should always elect to make things as obvious and unsurprising as possible.